app/views/hubstats/pull_requests/index.html.erb

```diff
@@ -10,8 +9,9 @@
         <input class="select2-control" id="users">
         </input>
       </div>
-
+      
```

Comment:
I'm really into the whole organization of the code makes sense, so I like to add whitespace to separate parts, but remove whitespace where it doesn't make sense.